### DISCUSSION TITLE
Creating a Searchable Reading List with Strapi CMS Custom API | njfamirm

### DISCUSSION THREAD

**Discussion by giscus** [njfamirm/blog#213] [Blog]
# Creating a Searchable Reading List with Strapi CMS Custom API | njfamirm

In this article, one of our big problems is investigated and to solve it, we create a reading-list with Strapi and make them searchable with Strapi custom API.

https://www.njfamirm.ir/en/blog/strapi-custom-api/

**Comment by njfamirm** [comment]
Thanks @derrickmehaffy for improve this article.
